ovos_translate_server/__main__.py

```
@@ -10,20 +10,34 @@
+    )
+    parser.add_argument(
+        "--tx-engine",
+        required=True,
+        help="OPM translation plugin entry-point name, e.g. ovos-translate-plugin-nllb",
+    )
+    parser.add_argument(
+        "--detect-engine",
+        default=None,
+        help="OPM language-detection plugin entry-point name (optional)",
+    )
+    parser.add_argument("--host", default="0.0.0.0", help="Host to bind (default: 0.0.0.0)")
```

Ruff (0.15.6)

[error] Possible binding to all interfaces

(S104)